packages/loot-core/src/client/query-hooks.tsx

```diff
@@ -23,7 +25,9 @@ function makeContext(queryState, opts, QueryClass) {
     useEffect(() => {
       if (query.getNumListeners() !== 0) {
         throw new Error(
-          'Query already has listeners. You cannot use the same query context `Provider` twice',
+          t(
```

Comment:
Same Q for these two.

Reply:
Both of these errors are thrown within components (at least `ManageRules.tsx`). The message would just be surfaced in the generic error handler for the app, like this:
<img width="560" alt="image" src="https://github.com/user-attachments/assets/7d881f9c-2725-4b5d-bec6-bb143ffd5dbe">

Reply:
Ah.. in my opinion - we should keep these in English.

Reply:
Gotcha. In that case, let's close this PR as it only contains unused translations 😅